drivers/ethernet/dsa_ksz8xxx.c

```diff
@@ -1065,7 +1066,8 @@ static struct dsa_api dsa_api_f = {
 	const struct dsa_slave_config dsa_0_slave_##slave##_config = {     \
 		.mac_addr = DT_PROP_OR(slave, local_mac_address, {0})      \
 	};                                                                 \
-	NET_DEVICE_DT_DEFINE_INSTANCE(slave,                               \
+	NET_DEVICE_INIT_INSTANCE(CONCAT(dsa_slave_port_, slave),           \
+	"lan" STRINGIFY(n),                                                \
```

Comment:
indentation nit

Reply:
@decsny - Shall anybody else also approve this fix we well? I'm asking as there is no other reviewer assigned and I do need two of them to have it pulled.